zmon-controller-app/src/main/java/org/zalando/zmon/ZmonApplication.java

```diff
@@ -12,6 +19,21 @@
 @SpringBootApplication
 @ComponentScan(basePackages = {"org.zalando.zmon"})
 public class ZmonApplication {
+	@Bean
+	public io.opentracing.Tracer jaegerTracer() {
```

Comment:
Configurations and bean definitions must be put into `org.zalando.zmon.config` package (check `org/zalando/zmon/config/CorsConfiguration.java` for example).

Reply:
done